packages/core-graphql/__tests__/graphql.test.js

```diff
@@ -1,51 +1,146 @@
 const app = require('./__support__/setup')
 
+/**
+ * Core plugins used by the testing suite.
+ */
 let graphql
 let logger
 
+/**
+ * Hard coded queries to run, data to feed into the database
+ * and expected results from the queries.
+ */
 const { testQueries,
         testData,
-        testExpected,
-        testHistory
+        testExpected
       } = require('./__fixtures__')
 
+/**
+ * Important elements to setUp and assign before running the tests.
+ * We setup a container (see __support__/) and grant access to
+ * both the GraphQL and Logger Core plugins to the suite.
+ */
 beforeAll(async () => {
   const container = await app.setUp()
   graphql = await container.resolvePlugin('graphql')
   logger = await container.resolvePlugin('logger')
   logger.info('Starting GraphQL Tests!')
 })
 
+/**
+ * Tear down the container.
+ */
 afterAll(() => {
   app.tearDown()
 })
 
+/**
+ * Ensure that our graphql plugin has been properly loaded
+ * then run the tests.
+ */
 describe('GraphQL', () => {
   it('should be an object', () => {
     expect(graphql).toBeObject()
   })
 
+  /**
+   * Test queries to be performed on the test data in the database
+   * and matched against expected results
+   */
   describe('test queries', () => {
     it('should be an object', () => {
       expect(testQueries).toBeObject()
+      logger.info('Processing test queries')
+    })
+
+    describe('test query for Block', () => {
+      it('Should have queries', () => {
+        expect(testQueries.block.length).toBeTruthy()
+
+        logger.debug('Proceeding with Block test queries')
+
+        testQueries.block.forEach((block) => {
+          logger.info(`Processing Block test query with id: ${block.id}`)
+        })
+      })
+    })
+
+    describe('test query for Blocks', () => {
+      it('Should have queries', () => {
```

Comment:
`it('Should` should always be `it('should` so it reads like a sentence and is consistent with all other occurrences of `it('should` throughout tests.